docs/template_plugin/src/template_executable_network.cpp

```diff
@@ -74,13 +65,14 @@ void TemplatePlugin::ExecutableNetwork::CompileGraph() {
         _inputIndex.emplace(parameter->get_friendly_name(), _function->get_parameter_index(parameter));
     }
 
-    // Perform any other steps like allocation and filling device buffers, and so on
+    // Perform any other steps like allocation and filling device buffers for weights, and so on
```

Comment:
Seams that `device buffers` should be replaced with `backend specific memory handles'

Reply:
fixed